examples/cluster/plot_color_quantization.py

```diff
@@ -91,12 +85,12 @@ def recreate_image(codebook, labels, w, h):
 plt.figure(2)
 plt.clf()
 plt.axis('off')
-plt.title('Quantized image (64 colors, K-Means)')
+plt.title('Quantized image (%d colors, K-Means)' % n_colors)
```

Comment:
```suggestion
plt.title(f'Quantized image ({n_colors} colors, K-Means)' )
```

Reply:
I just wanted to keep the code backward compatible with python versions prior to 3.6.
If that's not a problem, this change is perfectly OK.

Reply:
We support only Python 3.6+

Reply:
OK,I updated the code with f-strings